### PR TITLE
Request api endpoints

### PR DESCRIPTION
**Pull request by takeller**
## What's this PR do? 
- Adds request CRUD endpoints (create, get_all, get_one, update, destroy) 
- Updates Campaign with dependent destroy for associated requests

## Where should the reviewer start?
- src/views/RequestView.py

## How should this be manually tested? 
Fire up the server and start hitting those endpoints. 

## Background context? 
- Error handling needs to be improved, but that will be a future step 
- I changed the database relationships to add the dependent destroy. I removed backref from Request and put a db.relationship on Campaign

### CODE REVIEW

File: eth_crowdfund_be/eth_crowdfund_api/src/models/Campaign.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Campaign(db.Model):
   expiration = db.Column(db.DateTime)
   created_at = db.Column(db.DateTime)
   updated_at = db.Column(db.DateTime)
-  # requests = db.relationship('Request', backref='campaigns')
+  requests = db.relationship('Request', cascade="all, delete-orphan")
```
Comment (takeller) [inline eth_crowdfund_be/eth_crowdfund_api/src/models/Campaign.py:22]: Dependent Destroy

Comment (perryr16) [reply eth_crowdfund_be/eth_crowdfund_api/src/models/Campaign.py:22]: nice, good find

File: eth_crowdfund_be/eth_crowdfund_api/src/models/Request.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ class Request(db.Model):
   approvals = db.Column(db.Integer)
   created_at = db.Column(db.DateTime)
   updated_at = db.Column(db.DateTime)
-  campaign = db.relationship('Campaign', backref='requests')
+  campaign = db.relationship('Campaign')
```
Comment (perryr16) [inline eth_crowdfund_be/eth_crowdfund_api/src/models/Request.py:24]: no more backrefs?

File: eth_crowdfund_be/eth_crowdfund_api/src/views/RequestView.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+from flask import request, json, Response, Blueprint, g, jsonify
+from ..models.Campaign import Campaign, CampaignSchema
```
Comment (perryr16) [inline eth_crowdfund_be/eth_crowdfund_api/src/views/RequestView.py]: Is this line needed?

**Review by perryr16 — APPROVED** [review]

File: eth_crowdfund_be/eth_crowdfund_api/src/models/Request.py (hunk 2 above)
Comment (takeller) [reply eth_crowdfund_be/eth_crowdfund_api/src/models/Request.py:24]: Yeah. backref wasn't cooperating with the dependent destroy. Give me a sec and I'll test if we can still grab a Campaign through its Request

File: eth_crowdfund_be/eth_crowdfund_api/src/views/RequestView.py (hunk 3 above)
Comment (takeller) [reply eth_crowdfund_be/eth_crowdfund_api/src/views/RequestView.py]: Good catch. Don't think so

File: eth_crowdfund_be/eth_crowdfund_api/src/models/Request.py (hunk 2 above)
Comment (takeller) [reply eth_crowdfund_be/eth_crowdfund_api/src/models/Request.py:24]: Just confirmed that we can still grab a campaign through its request

**Review by AntonioJacksonII — APPROVED** [review]
Looks great! Thanks for finishing those last three endpoints.

**Pull request merged**